packages/experimental-loader/test/testkit/manifest-plugin.ts

```diff
@@ -0,0 +1,32 @@
+import { relative } from 'path';
+import { Module, Compiler, NormalModule, sources } from 'webpack';
+
+const isNormalModule = (module: Module): module is NormalModule => {
+    return (module as NormalModule).resource !== undefined;
+};
+
+export class TestManifestPlugin {
+    apply(compiler: Compiler) {
+        compiler.hooks.compilation.tap(TestManifestPlugin.name, (compilation) => {
+            compilation.hooks.additionalAssets.tap(TestManifestPlugin.name, () => {
+                const data: Record<string, string> = {};
+                for (const module of compilation.modules) {
+                    if (isNormalModule(module)) {
```

Comment:
change this `isNormalModule` to `isStylableModule` (or move resource.endsWith('.st.css') to the first `if`) and assume that you always have the `stylableNamespace` on the buildInfo. if you don't want to assume that you can throw instead